python/revng/daemon/schema.py

```diff
@@ -343,7 +344,7 @@ async def step_analysis_handle(_, info, **kwargs):
 schema_gen = SchemaGen()
 
 
-class SchemafulManager(Manager):
+class SchemafulManager(ManagerProxy):
```

Comment:
Reworked
Now there's a `ListenableManager` that has `add_event_listener` (I prefered to subclass it, since Manager is already quite big and handles the single responsability of "wrapping the C api")
This is then used on the `daemon` side to implement the fifos